src/settings-ui/Settings.UI/Assets/Settings/Scripts/InstallPowerShell7.ps1

```diff
@@ -0,0 +1,68 @@
+
+if ((Get-AppxPackage microsoft.DesktopAppInstaller).Version -ge [System.Version]"1.21")
+{
+  Write-Host "Detected winget. Will try to install PowerShell."
+}
+else
+{
+  Write-Host "WinGet not detected. Will try to install."
+  $cpuArchitecture="x64"
+  $detectedArchitecture=""
+  if ($env:PROCESSOR_ARCHITEW6432 -eq $null) {
+    $detectedArchitecture=$env:PROCESSOR_ARCHITECTURE
+  } else {
+    $detectedArchitecture=$env:PROCESSOR_ARCHITEW6432
+  }
+  Write-Host "Detected the CPU architecture:$detectedArchitecture"
+  if ($detectedArchitecture -ne "AMD64")
+  {
+    Write-Host "Mismatch with AMD64, setting it to arm64, since that's where we're likely running."
+    $cpuArchitecture="arm64"
+  }
+  if((Get-AppxPackage Microsoft.VCLibs.140.00).Version -ge [System.Version]"14.0.30704")
+  {
+    Write-Host "Detected Microsoft.VCLibs.140.00."
+  }
+  else
+  {
+    Write-Host "Microsoft.VCLibs.140.00 not detected. Will try to install."
+    Add-AppxPackage -Path "https://aka.ms/Microsoft.VCLibs.$cpuArchitecture.14.00.Desktop.appx"
+  }
+  if((Get-AppxPackage Microsoft.VCLibs.140.00.UWPDesktop).Version -ge [System.Version]"14.0.30704")
+  {
+    Write-Host "Detected Microsoft.VCLibs.140.00.UWPDesktop"
+  }
+  else
+  {
+    Write-Host "Microsoft.VCLibs.140.00.UWPDesktop not detected. Will try to install."
+    Add-AppxPackage -Path "https://aka.ms/Microsoft.VCLibs.$cpuArchitecture.14.00.Desktop.appx"
```

Comment:
this link is the same as line 29? should that be that way?

Reply:
Yes 🤷 That link seems to install both of those, so we just do it that way.